doc/faq.rst

```diff
@@ -22,25 +22,39 @@ need to install the ``virtualenv`` package, and then create a new virtual enviro
     virtualenv --python=python3 ~/eos
 
 
-The new virtual environment will now be activated.
+The new virtual environment will now be **activated**.
+That means, the `python` (and `pip`) command will now execute the local binary instead of a system-wide one.
 At any later point, you can (re-)activate the virtual environment by running the following command:
 
 ::
 
-    ~/eos/bin/activate
+    source ~/eos/bin/activate
 
-After activation, you can use install a binary distribution of EOS using ``pip3 install eoshep``.
+
+While activated, you can **install** a binary distribution of EOS using ``pip install eoshep``.
 If you plan to install EOS from source, as discussed in the :ref:`installation section<installation-from-source>`, you must set the prefix
 to match the path to your virtual environment, here: ``~/eos``.
 
-To deactivate the virtual environment, run
+You most likely want to install **jupyter and notebook**.
```

Comment:
[...] install the packages ``jupyter`` and ``notebook``.

Let's highlight that these are packages?

Reply:
```**`package`**``` does not render nicely in sphinx. I prefer to keep them as bold keywords. Do you agree?

Reply:
Ah, I see. No double formatting, right? Keep it as it is!